web/posts/2018-07-29-how-i-use-stuff-day-one.markdown

```diff
@@ -0,0 +1,184 @@
+---
+title: "How I Use Stuff: Day One"
+tags:
+---
+
+* App: [Day One](http://dayoneapp.com/)
+* Cost:  Free with a [$34.99/year Pro plan](http://help.dayoneapp.com/day-one-2-0/day-one-pricing-features-guide) (at the time of this writing)
+
+My journaling workflow has evolved a _lot_ over the past few years. My very
+first entry in Day One is from June 21, 2012, but I didn't start keeping a
+regular journal till early November 2015, which was shortly after I [quit
```

Comment:
What do you think about removing `which was`? I tend to remove words that I don't think add anything. Remove at your discretion.